src/Pact/Gas/Table.hs

```diff
@@ -169,9 +170,17 @@ defaultGasTable =
    ,("create-pact-guard",   const 1)
    ,("create-user-guard",   const 1)
    ,("enforce-guard",       const 11)
+   ,("install-capability",  const 1)
    ,("keyset-ref-guard",    const 1)
    ,("require-capability",  const 1)
    ,("with-capability",     const 2)
+
+   -- Constants and deprecated functions
+   ,("CHARSET_ASCII",       const 1)
```

Comment:
We had discussed not charging for constants/schemas -- I don't believe these are charged anyway, so we shouldn't have them.

Deprecated functions should just be sorted with everything else. We should just ditch `list`

@emilypi do we still support `verify-spv`, as in does it even work on chainweb? If not we should kill it. Can always add it later.

Reply:
We still support it, but nothing depends on it so it's very easy to kill that code. We just need to change `SPVSupport` to be single function. Then when we bump pact, we can just remove the `verifySPV` work from `Chainweb.Pact.SPV`

Reply:
Should I remove `verifySPV` from the Gas Table then?

Reply:
I deleted the constants/schemas and list from the gas table in latest commit.